### PR TITLE
W 11673765 processautomation managecredentials se

### PR DESCRIPTION
**Pull request by sebermuley**
restructured Credential Management, added Editing, Usage and Deletion

### CODE REVIEW

**Review by jeberhardST — DISMISSED** [review]

File: rpa-manager/modules/ROOT/pages/processautomation-prepare.adoc [hunk 1]
```diff
@@ -219,14 +219,80 @@ Delete global variables that will not be used anymore. Variables linked in confi
...
+* *AWS Credentials*
++
+for accessing Amazon Web Services
+* *User Account Credentials*
++
+for accessing all other kinds of applications
+
+You can link credentials anywhere that a globe symbol is displayed. A link symbol indicates an existing link. You can change and delete links.
+
+Editing or deleting a credential affects all configurations which use the credential.
```
Comment (dhirsch1) [inline rpa-manager/modules/ROOT/pages/processautomation-prepare.adoc:237]:
```suggestion
Editing or deleting a credential affects all configurations that use the credential.
```

File: rpa-manager/modules/ROOT/pages/processautomation-prepare.adoc [hunk 2]
```diff
@@ -219,14 +219,80 @@ Delete global variables that will not be used anymore. Variables linked in confi
...
+* *Type*:
++
+Type of the global credential. The type of credential determines where it can be used. The original value defined in RPA Builder is typed. Select one of the following types:
++
+** *AWS Credentials*:
++
+The credential is used for accessing Amazon Web Services. Fill in client ID, client secret and region. Using a session token is optional.
+** *User Account Credentials*
++
+The credential is used for accessing all other kinds of applications. Fill in the username and password
```
Comment (dhirsch1) [inline rpa-manager/modules/ROOT/pages/processautomation-prepare.adoc:268]:
```suggestion
The credential is used for accessing all other kinds of applications. Fill in the the username and password.
```

File: rpa-manager/modules/ROOT/pages/processautomation-prepare.adoc [hunk 3]
```diff
@@ -219,14 +219,80 @@ Delete global variables that will not be used anymore. Variables linked in confi
...
++
+The credential can be deployed in the production phase.
+
+* *Type*:
++
+Type of the global credential. The type of credential determines where it can be used. The original value defined in RPA Builder is typed. Select one of the following types:
++
+** *AWS Credentials*:
++
+The credential is used for accessing Amazon Web Services. Fill in client ID, client secret and region. Using a session token is optional.
```
Comment (dhirsch1) [inline rpa-manager/modules/ROOT/pages/processautomation-prepare.adoc:265]:
```suggestion
The credential is used for accessing Amazon Web Services. Fill in the client ID, client secret and region. Using a session token is optional.
```

File: rpa-manager/modules/ROOT/pages/processautomation-prepare.adoc [hunk 4]
```diff
@@ -219,14 +219,80 @@ Delete global variables that will not be used anymore. Variables linked in confi
...
+The credential is used for accessing all other kinds of applications. Fill in the username and password
 . Click *OK*.
 
+==== Check the Usage of a Credential
+
+Before editing a credential, check its usage to avoid inadvertent side-effects.
+
+. In the *Credential Pool* view of the *Process Automation* module, click the *Usage* icon image:usage-icon.png[binoculars symbol,1.5%,1.5%] in the table row of the credential you want to check.
+
+A window with a table shows you in which configurations the credential is used.
```
Comment (dhirsch1) [inline rpa-manager/modules/ROOT/pages/processautomation-prepare.adoc:277]: Suggestion:
A window with a table shows you the configurations in which the credential is used.

File: rpa-manager/modules/ROOT/pages/processautomation-prepare.adoc [hunk 5]
```diff
@@ -219,14 +219,80 @@ Delete global variables that will not be used anymore. Variables linked in confi
 . In the *Global Variables* view of the *Process Automation* module, click the *Delete* icon image:delete-icon.png[trash symbol,1.5%,1.5%] in the table row of the global variable you want to delete.
 . Confirm the deletion.
 
-=== Create Credentials
+=== Manage Credentials
 
 Credentials store login data for applications or web services. They can be used with different processes. Passwords are encrypted.
 
+The following types of credentials can be created:
```
Comment (dhirsch1) [inline rpa-manager/modules/ROOT/pages/processautomation-prepare.adoc:226]: A general suggestion for future writing - use active voice when possible; for example:

You can create the following types of credentials.

**Review by dhirsch1 — DISMISSED** [review]

**Review by Cristian-Venticinque — DISMISSED** [review]

**Review by Cristian-Venticinque — APPROVED** [review]

**Pull request merged**